DFC.ServiceTaxonomy.CompUi/Handlers/CacheHandler.cs

```diff
@@ -232,4 +327,19 @@ private Processing GetProcessingData(ContentContextBase currentContext, string p
 
         return null;
     }
+
+    //Temp method to determine if page route matches what should currently be processed.
+    //Can be removed once all routes are fine to be added.
+    private bool PageRouteFlag(Processing processing)
+    {
+        //Temp check to filter pages by certain allowed url's/titles
+        var current = JsonConvert.DeserializeObject<ContentItem>(processing.CurrentContent);
+        bool isAllowedPage = true;
+        if (processing.ContentType == "Page" && !eventGridAllowedPages.Any(pageUrl => current.PageLocationParts.FullUrl.Contains(pageUrl)))
```

Comment:
Instead of using a string, use the ContentTypes.Page as seen in the `switch` statement above.

```suggestion
        if (processing.ContentType == nameof(ContentTypes.Page) && !eventGridAllowedPages.Any(pageUrl => current.PageLocationParts.FullUrl.Contains(pageUrl)))
```